### PR TITLE
Use :migration_primary_key in create/1

### LINKED ISSUE #171: Migrator ignores primary key configuration without do block
### Environment

* Elixir version (elixir -v): 1.9.4
* Database and version (PostgreSQL 9.4, MongoDB 3.2, etc.): psql 12
* Ecto version (mix deps): 3.2.x, 3.3.x
* Database adapter and version (mix deps): postgrex 0.15.1
* Operating system: macOS Catalina

### Current behavior

`config :app, App.Repo, migration_primary_key: [type: :binary_id]`

`$ mix ecto.gen.migration create_foo`

```elixir
create table(:foo, prefix: "cms")
```

```
foo_dev# \d cms.foo
                               Table "cms.foo"
 Column |  Type  | Collation | Nullable |                  Default
--------+--------+-----------+----------+--------------------------------------------
 id     | bigint |           | not null | nextval('cms.foo_id_seq'::regclass)
```

Oddly:

```elixir
create table(:foo, prefix: "cms") do
end
```

```
foo_dev# \d cms.foo
                               Table "cms.foo"
 Column | Type | Collation | Nullable | Default
--------+------+-----------+----------+---------
 id     | uuid |           | not null |
```

### Expected behavior

A `do` block shouldn't be required to get a primary key of type `uuid`.

### PR DESCRIPTION
**Pull request by wojtekmach**
Closes #171

### CODE REVIEW

File: lib/ecto/migration.ex [hunk 1]
```diff
@@ -528,7 +527,11 @@ defmodule Ecto.Migration do
   defp do_create(table, command) do
     columns =
       if table.primary_key do
-        [{:add, :id, :bigserial, primary_key: true}]
+        opts = Runner.repo_config(:migration_primary_key, [])
+        opts = Keyword.put(opts, :primary_key, true)
+        {name, opts} = Keyword.pop(opts, :name, :id)
+        {type, opts} = Keyword.pop(opts, :type, :bigserial)
```
Comment (wojtekmach) [inline lib/ecto/migration.ex]: these 4 lines are duplicated (see the top of this diff) but not sure if it's worth extracting out.

Comment (wojtekmach) [reply lib/ecto/migration.ex]: I refactored this in 78f876b.

File: lib/ecto/migration.ex [hunk 2]
```diff
@@ -1189,4 +1185,14 @@ defmodule Ecto.Migration do
...
+
+  @doc false
+  def __primary_key__() do
+    opts = Runner.repo_config(:migration_primary_key, [])
+    opts = Keyword.put(opts, :primary_key, true)
+    {name, opts} = Keyword.pop(opts, :name, :id)
+    {type, opts} = Keyword.pop(opts, :type, :bigserial)
+    {name, type, opts}
+  end
+
```
Comment (josevalim) [inline lib/ecto/migration.ex]:
```suggestion
```

Comment (wojtekmach) [reply lib/ecto/migration.ex]: yikes, thanks! :D

**Comment by whitepaperclip** [comment]
Wow, that was fast!

**Pull request merged**